### PR TITLE
Split transpiler tests, add KZG tests

### PR DESCRIPTION
**Pull request by ioxid**

### CODE REVIEW

File: transpiler/test/evm.cpp [hunk 1]
```diff
@@ -0,0 +1,1632 @@
...
+
+        return true;
+    }
+
+    plonk_table_description<field_type> desc;
+};
+
+BOOST_AUTO_TEST_SUITE(placeholder_circuit2_kzg_v2)
+
+using TestFixtures = boost::mpl::list<
```
Comment (ETatuzova) [inline transpiler/test/evm.cpp:1568]: Please, add nil::crypto3::algebra::curves::alt_bn128_254::base_field_type curve here.

**Review by ETatuzova — CHANGES_REQUESTED** [review]

File: transpiler/test/evm.cpp [hunk 2]
```diff
@@ -0,0 +1,1632 @@
...
+    const plonk_table_description<typename PlaceholderParams::field_type> &table_description,
+    std::string folder_name
+) {
+    std::filesystem::create_directory(folder_name);
+    test_placeholder_proof<Endianness, placeholder_proof<typename PlaceholderParams::field_type, PlaceholderParams>>(
+        proof,
+        commitment_scheme.get_commitment_params(),
+        folder_name + "/proof.bin"
+    );
+    test_placeholder_partial_proof<
```
Comment (ETatuzova) [inline transpiler/test/evm.cpp:386]: Remove it. Not used here

File: transpiler/test/evm.cpp (hunk 1 above)
Comment (ioxid) [reply transpiler/test/evm.cpp:1528]: Oh, right, it's the one with built-in support in EVM. Done.

**Comment by ioxid** [comment]
I found that verifier printer parameters in tests are probably incorrect, because the comments mention wrong parameters. This is probably a result of some refactoring. Since I don't know what these parameters mean I don't really know what should be there. But it can break in unexpected ways.
Screenshot_20241007_105457

(notice that 15 is passed for an optional boolean parameter, and two other parameter names don't match)

**Review by ETatuzova — APPROVED** [review]

**Review by ETatuzova — APPROVED** [review]

**Review by ETatuzova — APPROVED** [review]

**Review by ETatuzova — APPROVED** [review]

**Pull request merged**